modules/cloudsql-instance/variables.tf

```diff
@@ -143,6 +143,36 @@ variable "labels" {
   default     = null
 }
 
+variable "maintenance_config" {
+  description = "Set maintenance window configuration and maintenance deny period (up to 90 days). Date format: 'yyyy-mm-dd'."
+  type = object({
+    maintenance_window = optional(object({
+      day          = number
+      hour         = number
+      update_track = optional(string, null)
```

Comment:
we still need validation on this for canary/stable/null :)

Reply:
You're right, I missed it sorry. Now it should be fine!